### DISCUSSION TITLE
Laravel 8 Support in PHP 7.3

### DISCUSSION THREAD

**Discussion by eliurkis** [spatie/laravel-medialibrary#2117] [Q&A] [answered]
The version of the package that supports Laravel 8, requires PHP 7.4 to be installed. But I have Laravel 8 in PHP 7.3, what can I do?

![image](https://user-images.githubusercontent.com/59236/96378107-f7abf200-1157-11eb-8909-e732304d5bc3.png)

**Comment by freekmurze** [comment, marked as answer]
Either upgrade to PHP 7.4 or fork the package and make it compatible yourself with the PHP version you are using.

**Reply by eliurkis** [reply]
Thanks! Will do
